inc/class-wpseo-statistics.php

```diff
@@ -57,86 +57,4 @@ public function get_post_count( $rank ) {
 
 		return $posts->found_posts;
 	}
-
-	/********************** DEPRECATED METHODS **********************/
```

Comment:
I'm not sure if we want to remove this header as we might have deprecated methods in the feature.